component/http/middleware/middleware.go

```diff
@@ -122,6 +124,21 @@ func NewRecovery() Func {
 	}
 }
 
+// NewAppNameVersion returns to all responses a header which contain the app name and version.
```

Comment:
- It would be useful to clarify that it will also overwrite previous values
```suggestion
// NewAppNameVersion adds an app name header and an app version header to all responses. Existing values of these headers are overwritten
```